src/shared/services/request/request.options.builder.ts

```diff
@@ -46,7 +46,7 @@ export class RequestOptionsBuilder {
       if (caseCriteria) {
         for (let criterion of Object.keys(caseCriteria)) {
           if (RequestOptionsBuilder.includeParam(caseCriteria[criterion])) {
-            params = params.set(RequestOptionsBuilder.FIELD_PREFIX + criterion, caseCriteria[criterion].trim());
+            params = params.set(RequestOptionsBuilder.FIELD_PREFIX + criterion, caseCriteria[criterion].trim ? caseCriteria[criterion].trim() : caseCriteria[criterion]);
```

Comment:
Can we take the building of variables out of the function calls?

Reply:
Agree with Phil, If you are doing that you could write some tests for that function in isolation. Thanks